source/release-notes.txt

```diff
@@ -10,6 +10,48 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.20
+--------------
+
+*Released November, 2019*
+
+- New and improved :guilabel:`New Connection` user experience with
```

Comment:
I would just call this "New and improved Connection experience..." and link the word Connection to the Connect page.